runtime/onert/core/include/ir/ModelGraph.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_MODEL_GRAPH_H__
+#define __ONERT_IR_MODEL_GRAPH_H__
+
+#include <memory>
+
+#include "ir/Index.h"
+#include "ir/Models.h"
+#include "ir/Subgraphs.h"
+
+namespace onert
+{
+namespace ir
+{
+
+class ModelGraph
+{
+public:
+  ModelGraph() = default;
+  ModelGraph(const ModelGraph &obj) = delete;
```

Comment:
For https://github.com/Samsung/ONE/pull/9244#issuecomment-1155789340), copy constructor is deleted as `Models`'s copy constructor is deleted